clients/ui/frontend/src/app/AppRoutes.tsx

```diff
@@ -42,7 +42,7 @@ export const useAdminSettings = (): NavDataItem[] => {
 export const useNavData = (): NavDataItem[] => [
   {
     label: 'Model Registry',
-    path: '/',
+    path: '/modelRegistry',
```

Comment:
is it normal to do camel case for URLs?

Reply:
This is a valid point, im just porting midstream code but we can refactor in the future.

Reply:
yeah, agree with Griffin, but no need to change right now.